generators/app/constants.js

```diff
@@ -44,11 +44,11 @@ module.exports = {
   LICENSE_DEFAULT,
 
   PROMPT_DEFAULTS: {
-    docs: false,
-    css: false,
+    docs: true,
+    css: true,
```

Comment:
We can drop these changes here if we want, but I think that we want to offer the full toolset for --hurry.